source/_docs/continuous-integration.md

```diff
@@ -17,18 +17,69 @@ Continuous Integration (CI) is a method of running automated unit and integratio
-
-    SITE_NAME=yoursitename
-    ENV=dev
-    drush @pantheon.$SITE_NAME.$ENV test-run -l https://$ENV-$SITE_NAME.pantheon.io/ UserSaveTestCase --xml='sites/default/files'
-
-The end result is written to https://dev-yoursitename.pantheon.io/sites/default/files/UserSaveTestCase.xml
-
+## Drupal SimpleTest
+
+[SimpleTest](https://drupal.org/project/simpletest){.external} is a testing framework based on the [SimpleTest PHP library](https://github.com/simpletest/simpletest){.external} that is included with Drupal core. If you are creating a custom web application, you should consider including SimpleTests of your module functionality.
+
+[SiteTest](https://www.drupal.org/project/site_test){.external} is a contrib module designed to allow running tests directly against your sites code instead of a base Drupal clone of your site.  This module is recommended for use of SimpleTest on Pantheon.
+
+<div class="alert alert-info" role="alert" markdown="1">
+#### Note {.info}
+The drush test-run command was dropped in Drush 7 and 8. See [this GitHub issue](https://github.com/drush-ops/drush/issues/1362){.external} for more details.
+</div>
+
+To run tests on Pantheon:
+
+1. Enable `site_test`:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- en site_test -y
+   ```
+
+1. Clear the cache immediately before running test to avoid strange failures:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- cc all
+   ```
+
+1. Get the absolute path before you can run the script:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- eval "echo DRUPAL_ROOT"
+   ```
+
+  In the command above, you may need to strip out warnings by ending the command with `2>/dev/null`.
+
+The full command will look something like this:
+
+```bash
+terminus drush $SITE_NAME.$ENV_NAME -- exec php `terminus drush $SITE_NAME.$ENV_NAME -- eval "echo DRUPAL_ROOT" 2>/dev/null`/scripts/run-tests.sh --url http://$ENV_NAME-$SITE_NAME.pantheonsite.io OptionalTestGroup
+```
+
+In the above command the `--url` option is required to be passed as multidevs do not respond to `localhost`.
+
+A full CircleCI command might look similar to this:
+
+```
+      - run:
+          name: Test simpletest
+          command: |
+            if [ "${CIRCLE_BRANCH}" != "master" ]; then
+
+              terminus drush $SITE_NAME.$ENV_NAME -- en site_test -y
+
+              # If you don't clear the cache immediately before running tests
+              # we get the html gibberish instead of a passing test.
+              terminus drush $SITE_NAME.$ENV_NAME -- cc all
+
+              # NOTE: It is a requirement to use Terminus <=1.9
+              # in order to exclude the notice in shell output of the
+              # embedded command to find the absolute path.
+              terminus drush $SITE_NAME.$ENV_NAME -- exec php `terminus drush $SITE_NAME.$ENV_NAME -- eval "echo DRUPAL_ROOT" 2>/dev/null`/scripts/run-tests.sh --url http://$ENV_NAME-$SITE_NAME.pantheonsite.io ExampleTestGroupName
```

Comment:
@alexfornuto To be consistent lest change `ExampleTestGroupName` to `OptionalTestGroup`.  Maybe they should both read `OptionalTestGroupName`?

Reply:
Resolved by https://github.com/pantheon-systems/documentation/pull/4524/commits/68085c0872b0a730060406607284f7a18f0e8bf8